Contest_Archive_Format.md

```diff
@@ -16,708 +17,166 @@ disk, including:
-
-#### Examples
-
-```json
-{
-  "command": "gcc",
-  "args": "-O2 -Wall -o a.out -static {files}",
-  "version": "gcc (Ubuntu 8.3.0-6ubuntu1) 8.3.0"
-}
-```
-
-```json
-{
-  "command": "javac",
-  "args": "-O {files}",
-  "version": "javac 11.0.4",
-  "version-command": "javac --version"
-}
-```
-
-```json
-{
-  "command": "java",
-  "version": "openjdk version \"11.0.4\" 2019-07-16"
-}
-```
-
-### Problem object
-
-| Name            | Type    | Required | Description |
-| :-------------- | :------ | :------- | :---------- |
-| id              | string  | yes      | Identifier of the problem. Must match problem package name if present. |
-| uuid            | string  | depends  | UUID of the problem. Must be present if problem package is not present. |
-| label           | string  | no       | Label of the problem on the scoreboard, typically a single capitalized letter. |
-| name            | string  | no       | Name of the problem. |
-| ordinal         | integer | no       | Ordering of problems on the scoreboard. |
-| rgb             | string  | no       | Hexadecimal RGB value of problem color as specified in [HTML hexadecimal colors](http://en.wikipedia.org/wiki/Web_colors#Hex_triplet), e.g. '#AC00FF' or '#fff'. |
-| color           | string  | no       | Human readable color description associated to the RGB value. |
-| time_limit      | number  | no       | Time limit in seconds per test data set (i.e. per single run). |
-| test_data_count | integer | no       | Number of test data sets. |
-
-#### Differences from Contest API
-
-- The `time_limit` element is required to be a `number` that is an
-  integer multiple of `0.001` in the Contest API. 
-
-#### Examples
-
-```json
-{
-  "id": "asteroids",
-  "uuid": "cb1c4b77-b203-4943-a13f-9b89dec1ac11",
-  "label": "A",
-  "name": "Asteroid Rangers",
-  "ordinal": 1,
-  "rgb":"#00f",
-  "color": "blue",
-  "time_limit": 2,
-  "test_data_count": 10
-}
-```
-
-### Group object
-
-| Name    | Type    | Required | Description |
-| :------ | :------ | :------- | :---------- |
-| id      | ID      | yes      | Identifier of the group. |
-| icpc_id | string  | no       | External identifier from ICPC CMS. |
-| name    | string  | yes      | Name of the group. |
-| type    | string  | no       | Type of this group. |
-| hidden  | boolean | no       | If group should be hidden from scoreboard. Defaults to false if missing. |
-
-#### Differences from Contest API
-
-None.
-
-#### Examples
-
-```json
-{
-  "id": "asia",
-  "icpc_id": "7593",
-  "name": "Asia"
-}
+A package contains information regarding a single contest (corresponding to
+the `/contests/<id>/*` endpoints of the API). The API can contain information
+for several contests, to store all information a package per contest would be
+needed.
+
+Information in the API is always either in JSON format, [NDJSON]
+(contest_api#event-feed) format, or linked using a [file reference]
+(contest_api#json-attribute-types) JSON object.
```

Comment:
I think the line breaks between [NDJSON] and (contest_api#event-feed) and similar for [file reference] and (contest_api#json-attribute-types) are causing the markdown to render incorrectly.

Reply:
Fixed.